taca/storage/storage.py

```diff
@@ -114,3 +218,31 @@ def _send_to_swestore(f, dest, remove=True):
         else:
             LOG.info("Run {} is not {} days old yet. Not archiving".format(run, str(days)))
     os.remove("{}.archiving".format(run.split('.')[0]))
+
+
+def get_closed_projects(projs,pj_con,days,logger=None):
```

Comment:
yeah I think so, its more consistent with the rest of the code. 

On the other hand... I may think on a decorator for both LOG and config, I'm constantly loading them.